db/migrate/20170212055707_create_teams.rb

```diff
@@ -0,0 +1,16 @@
+class CreateTeams < ActiveRecord::Migration[5.0]
+  def change
+    create_table :teams do |t|
+      t.string :team_name
```

Comment:
I agree. Let's rename columns on the `users` table like you described and let's also shorten this table's `team_name` to `name`.